src/test/java/net/comments/sample/TrainingRoomLimitedTest.java

```diff
@@ -0,0 +1,25 @@
+package net.comments.sample;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.collection.IsIterableContainingInOrder;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TrainingRoomLimitedTest {
+
+    @Test
+    public void testGetEnteredPerson() {
```

Comment:
I don't see that you use `TrainingRoomLimited` here. Is this correct?

Reply:
@extsoft It's correct, because in this class I have only one method that returns value. Maybe another methods should be check in this class (e.g. "accept")? Or some parameters should be checked for this class(e.g. size)?

Reply:
Try to test something from `TrainingRoomLimited` class.